packages/manager/modules/pci/src/projects/project/storages/cold-archive/onboarding/onboarding.controller.js

```diff
@@ -0,0 +1,41 @@
+import reduce from 'lodash/reduce';
+import illustration from './assets/ColdStorage.png';
+import { TRACKING } from './onboarding.constants';
+import { GUIDES } from '../cold-archives.constants';
+
+export default class PciStorageColdArchivesOnboardingController {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.illustration = illustration;
+    this.guides = reduce(
+      GUIDES,
+      (list, guide) => [
+        ...list,
+        {
+          ...guide,
+          title: this.$translate.instant(
+            `pci_projects_project_storages_cold_archives_onboarding_guides_${guide.id}_title`,
+          ),
+          description: this.$translate.instant(
+            `pci_projects_project_storages_cold_archives_onboarding_guides_${guide.id}_description`,
+          ),
+        },
+      ],
+      [],
+    );
+  }
+
+  addColdArchive() {
+    this.trackClick(TRACKING.ONBOARDING_CREATE_USER);
+    this.goToAddColdArchive();
```

Comment:
why add the return statement ?

Reply:
We do that because `goToAddColdArchive` function return a promise.
So, it's a good practice to have a `promise` instead of `undefined`.

Also, if tomorrow for some raisons we have to use `addColdArchive` in another place we have the ability to chain new actions.